src/components/ThemeProvider/stories/full.md

```diff
@@ -43,20 +43,19 @@ function ExampleComponent(props) {
 
 | Property           | Description                                  | Type                                                           | Default |
 | ------------------ | -------------------------------------------- | -------------------------------------------------------------- | ------- |
-| `theme`            | A custom theme                               | `GearboxTheme`                                                 |         |
+| `theme`            | A custom theme                               | `GearboxTheme`, `GearboxThemeOptional`                         |         |
```

Comment:
Leave it as it was before just `GearboxTheme`

Reply:
Removed